packages/bpk-mixins/src/mixins/_typography.scss

```diff
@@ -718,6 +700,7 @@ $bpk-spacing-v2: true;
 ///     @include bpk-heading-1();
 ///     @include bpk-heading--no-bottom-margin();
 ///   }
+/// @deprecated Headings no longer set margins.
```

Comment:
This mixin is only used by bpk-component-heading and given that is deprecated we can deprecate this